src/GitVersion.Core/VersionCalculation/VersionStrategies.cs

```diff
@@ -0,0 +1,14 @@
+namespace GitVersion.VersionCalculation;
+
+[Flags]
+public enum VersionStrategies
+{
+    None = 0,
+    ConfigNext = 1,
+    MergeMessage = 2,
+    TaggedCommit = 4,
+    TrackReleaseBranches = 8,
+    VersionInBranchName = 16,
+    NonTrunkBased = ConfigNext | MergeMessage | TaggedCommit | TrackReleaseBranches | VersionInBranchName,
+    TrunkBased = 32,
```

Comment:
I think combined flag values should be the last value in the enum.

```suggestion
    TrunkBased = 32,
    NonTrunkBased = ConfigNext | MergeMessage | TaggedCommit | TrackReleaseBranches | VersionInBranchName,
```

Reply:
done